contracts/libraries/SafeToL2Migration.sol

```diff
@@ -96,7 +96,7 @@ contract SafeToL2Migration is SafeStorage {
             0,
             0,
             address(0),
-            address(0),
+            payable(address(0)),
```

Comment:
Is this change needed?

Reply:
Not anymore, it was edited for the optimizer, I think.

Reply:
It's needed again to make it compatible to be compiled with 0.8.x version.